src/main/java/com/github/phuonghuynh/compressor/Sample.java

```diff
@@ -0,0 +1,8 @@
+package com.github.phuonghuynh.compressor;
+
+public final class Sample {
+  
+  public void func() {
+     System.out.println("something here");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)